<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSensorConf</class>
 <widget class="QWidget" name="WidgetSensorConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temperature</string>
  </property>
  <property name="windowIcon">
   <iconset theme="configure">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="scaleCbox">
       <item>
        <property name="text">
         <string>Celsius</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fahrenheit</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="textRbt">
       <property name="text">
        <string>Te&amp;xt</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Display:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Update interval:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QRadioButton" name="barsRbt">
       <property name="text">
        <string>Bars</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="1">
            <widget class="QSpinBox" name="criticalsbox">
             <property name="suffix">
              <string> C</string>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Sensor:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBox"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Critical:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="warningsbox">
             <property name="suffix">
              <string> C</string>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Warning:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="3" column="1">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Bar Spacing:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="backcolorbt">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Display Height</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2" colspan="2">
            <widget class="QSpinBox" name="barspacingsbox">
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Background Color:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Max Temp:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2" colspan="2">
            <widget class="QSpinBox" name="barwidthsbox">
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Bar Width:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2" colspan="2">
            <widget class="QSpinBox" name="marginsbox">
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="QSpinBox" name="heightsbox">
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="5" column="2" colspan="2">
            <widget class="QSpinBox" name="maxtempsbox">
             <property name="suffix">
              <string> C</string>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Margin:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4" rowspan="7">
            <widget class="QTreeWidget" name="treeWidget">
             <column>
              <property name="text">
               <string>Enabled</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Sensor</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QSpinBox" name="spinBox">
       <property name="suffix">
        <string>s</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okbt">
         <property name="text">
          <string>Apply</string>
         </property>
         <property name="icon">
          <iconset theme="dialog-ok-apply">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelbt">
         <property name="text">
          <string>Close</string>
         </property>
         <property name="icon">
          <iconset theme="dialog-close">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Scale:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelbt</sender>
   <signal>clicked()</signal>
   <receiver>WidgetSensorConf</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
